### ISSUE TITLE
[TODO]: Create a plugin

### ISSUE THREAD

**Issue by C-likethis123** [C-likethis123/nhn-underline-plugin#1]
Previously, I have a custom fork of the TUI Editor that contained an underline button. This should be refactored into a plugin so others can integrate this feature into their projects.

**Child issues:**
- [x] #3 
- [ ] #4 

# Resources

- My PR, so you can refer to what I have done previously: https://github.com/C-likethis123/tui.editor/pull/1
- NHN's Colour Syntax plugin, which is the most similar to what we are doing: https://github.com/nhn/tui.editor/tree/master/plugins/color-syntax
- NHN's documentation on creating plugins: https://nhn.github.io/tui.editor/latest/tutorial-example17-creating-plugin

**Comment by maca-sys** [comment]
So we have to implement the added code in your PR to a js file?

**Comment by C-likethis123** [comment]
That's right, you'll have to adapt parts of my code to a JS file similar to the structure of the colour syntax plugin (https://github.com/nhn/tui.editor/blob/master/plugins/color-syntax/src/js/index.js).

In my custom fork, I implemented the underline feature by wrapping the text around with `<u></u>` tags. The colour syntax plugin does something similar, it wraps the text around with `<span></span>` tags. I think you can play around with the colour syntax plugin and see what it does, before adapting it to the underline plugin.

**Comment by C-likethis123** [comment]
Hi @masoudtahmasebi93, how's your progress on the plugin so far? If you have started work on the task, do you need any more pointers or resources to complete the task? If not, do you mind if I do it instead?
